manifests/init.pp

```diff
@@ -377,6 +379,7 @@
   Optional[Integer[0]] $dhcp_mclt = $::foreman_proxy::params::dhcp_mclt,
   Optional[Integer[0, 255]] $dhcp_load_split = $::foreman_proxy::params::dhcp_load_split,
   Optional[Integer[0]] $dhcp_load_balance = $::foreman_proxy::params::dhcp_load_balance,
+  Optional[Boolean] $dhcp_manage_acls = $::foreman_proxy::params::dhcp_manage_acls,
```

Comment:
Optional is only needed if you want to permit undef. Given the default is already a boolean Boolean should suffice.